pages/wards/[id]/schedule-visit.js

```diff
@@ -253,8 +253,8 @@ const queryContainsInitialData = (query) => {
 
 export const getServerSideProps = propsWithContainer(
   verifyToken(
-    async ({ query, container }) => {
-      const { id } = query;
+    async ({ query, container, authenticationToken }) => {
+      const id = authenticationToken.ward;
```

Comment:
Should we call this wardId so it's clear what it relates to ?

Reply:
We're actually going to remove the need for this in a next PR, so will kindly skip this